src/c2goto/library/builtin_libs.c

```diff
@@ -28,6 +28,8 @@
 
 #    include <cheri/cheric.h>
 
+__SIZE_TYPE__ get_all_permissions(const cc128_cap_t *cap);
```

Comment:
Shouldn't this function be declared with two leading underscores?
Alternatively, shouldn't it be inside the cheri ifdef?